app/models/setting.rb

```diff
@@ -283,13 +278,18 @@ def self.load_defaults
     true
   end
 
+  def collection_method
```

Comment:
Would this sound better? :thinking: 
```suggestion
  def collection_method_name
```